reg/drone/service/sensor/battery/FaultFlags.0.1.uavcan

```diff
@@ -0,0 +1,39 @@
+# A collection of detailed fault flags indicating problems detected by the service provider.
+# A fault flag is set when the corresponding parameter exceeds its safe operating area (SOA) as defined by the vendor;
+# see https://en.wikipedia.org/wiki/Safe_operating_area.
+# As long as at least one flag is set, the service health should not be NOMINAL.
+
+bool temperature_hot
+bool temperature_cold
+# The temperature of at least one cell/section of the battery or the battery electronics is above or below
+# the safe operating limits. Note that the flags are not mutually exclusive.
+
+bool overload
+# The charge or discharge rate exceeds the safe operating limits.
+
+bool overdischarge
+# The battery has been discharged beyond the design limits and may have incurred damage.
+# This flag would usually be set in combination with needs_service.
+
+bool bad_battery
+# The battery should not be used anymore.
+
+bool needs_service
+# The battery requires offline maintenance.
+
+bool bms_error
+# Internal error in the battery management system, not related to the battery itself.
+
+bool balancer
+# The battery balancing circuits are malfunctioning.
+
```

Comment:
This is not a common error flag. This flag can be rather included in `bms_error` flag. There may be different vendor specific errors concerning balancer malfanctioning (no balance current, balanse resistors/pcbs overheat).

Reply:
Removed.